File: Server/Toopeto/JsonPacket/MessagePacket.cs

using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using AnarSoft.Utility.Utilities;
namespace Toopeto.JsonPacket
{
/**
 * Created by mehrang on 03/03/2015.
 */
public class MessagePacket : Json {

       [DataMember]
    public string data;

    public MessagePacket():base()
    {
        
        this.packetName = PacketNameEnum.message.ToString();
       
    }

  public override string getString()  {
        return JsonSerializer.Serialize(this);
    }
  public static MessagePacket GetMessagePacket(string jsonString)
  {
      return JsonSerializer.DeSerialize<MessagePacket>(jsonString);
  }
  
}

}

File: ClientAndroid/src/main/java/ir/bilgisoft/toopeto/json/MessagePacket.java
package ir.bilgisoft.toopeto.json;

import org.json.JSONException;
import com.google.gson.Gson;

import ir.bilgisoft.toopeto.entities.Account;

/**
 * Created by mehrang on 03/03/2015.
 */
public class MessagePacket extends Json {


    public String data;

    public MessagePacket()
    {
        super();
        this.packetName =Enums.PacketNameEnum.message.toString();
    }




    @Override
    public String getString() {
        Gson gson=new Gson();
        return gson.toJson(this);
    }


    public static MessagePacket GetMessagePacket(String stringJson)  {
        Gson gson = new Gson();
        MessagePacket message=gson.fromJson(stringJson,MessagePacket.class);
        return  message;

    }


}
